src/include/txn.i

```diff
@@ -965,15 +965,16 @@ __wt_txn_id_alloc(WT_SESSION_IMPL *session, bool publish)
 
 	/*
 	 * Allocating transaction IDs involves several steps.
+	 * (Note that this requires proof-of-concept now)
 	 *
-	 * Firstly, we do an atomic increment to allocate a unique ID.  The
-	 * field we increment is not used anywhere else.
+	 * Firstly, publish the current transaction ID to the global transaction
```

Comment:
the comment needs updating, looks like you have moved from an invalid ID to a flag.